pkg/crypto/crypto.go

```diff
@@ -5,14 +5,17 @@ import (
 	"io"
 	"io/ioutil"
 	"strings"
-
+	
 	"github.com/pkg/errors"
 	"golang.org/x/crypto/openpgp"
 	"golang.org/x/crypto/openpgp/armor"
 )
 
 const (
 	wmcoMarker = "<wmcoMarker>"
+	startTag = "-----BEGIN ENCRYPTED DATA-----"
```

Comment:
Do these constants relate to the [tag defined here](https://github.com/openshift/windows-machine-config-operator/pull/748/files#diff-8a03969770d7b59692a408c8f8cfcff4c2b4f21281ac81f36d94d4b39306c434R46)?